src/Compilers/CSharp/Test/CommandLine/CommandLineTests.cs

```diff
@@ -12985,11 +12987,13 @@ class C
 
             analyzerConfig = analyzerConfigFile.WriteAllText(@"
 is_global = true
+global_level = 100
 [/file.cs]
 option1 = abc");
 
             analyzerConfig2 = analyzerConfigFile2.WriteAllText(@"
 is_global = true
+global_level = 100
```

Comment:
Why did we need to add these entries here? Didn't see any change to the tests verifying the text here so not sure why we needed to add this.

Reply:
These tests are checking the error messages that occur when keys collide, so I made it explicit that they would continue to collide. I could rename the files and let the heuristic pick up the 0 value, but I figured it was better to be explicit in the test.

Reply:
Ah okay, makes sense.